deployment/arm/global.json

```diff
@@ -15,9 +15,63 @@
             "defaultValue": [
             ],
             "type": "array"
-        }
+        },
+        "cosmosdbName":{
```

Comment:
It's still strange the the whitespace is showing up off here.  Try to run an auto-format on the JSON file in vs code.  I also use a VS code extension `Azure Resource Manager (ARM) Tools` extension that gives some better intelligence for ARM template development.

Reply:
Maybe something similar to how I am using `frontendHosts` and `backendHosts`.  They are passed in as arrays.

Reply:
I already have these extension installed but there is some weirdness with my environment that .json are not well formatted when sent out for PR. However, It shows all formatted on my IDE. Debugging this.